blockscout-ens/bens-server/src/settings.rs

```diff
@@ -28,10 +28,49 @@ impl ConfigSettings for Settings {
     const SERVICE_NAME: &'static str = "BENS";
 }
 
+// TODO: move database settings to blockscout-service-launcher
 #[derive(Debug, Clone, Deserialize, PartialEq, Eq)]
 #[serde(deny_unknown_fields)]
 pub struct DatabaseSettings {
-    pub url: String,
+    pub connect: DatabaseConnectSettings,
+}
+
+#[derive(Debug, Clone, Deserialize, PartialEq, Eq)]
+#[serde(deny_unknown_fields, rename_all = "lowercase")]
+pub enum DatabaseConnectSettings {
+    Url(String),
+    KV(DatabaseKVConnection),
+}
+
+impl DatabaseConnectSettings {
+    pub fn url(&self) -> String {
```

Comment:
Nit: I believe receiving the argument by value may better convey the intentions. That are, to convert the temporal connection string values to the url

Reply:
ok